### PR TITLE
[1.12] feat: add IsExternalURL doc

### LINKED ISSUE #989: [Enhancement] Add IsExternalURL doc
### Description

Add doc for `is_external_url()` JMESPath custom filter.

### Slack discussion

_No response_

### PR DESCRIPTION
**Pull request by UgOrange**
## Related issue #

Closes #989 

## Proposed Changes
Add doc for `is_external_url()` JMESPath custom filter.

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue. #989

### CODE REVIEW

**Comment by UgOrange** [comment]
Hi, @chipzoller 

PR [#8614](https://github.com/kyverno/kyverno/pull/8614) has been merged. Can you please review and update the documentation with these changes?

Thanks.

**Review by chipzoller — APPROVED** [review]
Looks good, thank you. But since the 1.11 release branch has still not been cut, we'll hold this until that point, then merge to `main`.

File: content/en/docs/Writing policies/jmespath.md [hunk 1]
```diff
@@ -2963,3 +2963,42 @@ spec:
...
+    - name: validate-external-url
+      match:
+        resources:
+          kinds:
+            - ConfigMap
+      validate:
+        message: "ConfigMap contains an external URL."
+        deny:
+          conditions:
+            - key: "{{ request.object.data.test | is_external_url(@) }}"
```
Comment (chipzoller) [inline content/en/docs/Writing policies/jmespath.md:2998]:
```suggestion
           all:
             - key: "{{ request.object.data.test | is_external_url(@) }}"
```

File: content/en/docs/Writing policies/jmespath.md [hunk 2]
```diff
@@ -2963,3 +2963,44 @@ spec:
...
+kind: ClusterPolicy
+metadata:
+  name: check-external-url-in-configmap
+spec:
+  validationFailureAction: enforce
+  background: false
+  rules:
+    - name: validate-external-url
+      match:
+      any:
```
Comment (chipzoller) [inline content/en/docs/Writing policies/jmespath.md:2991]:
```suggestion
        any:
```

File: content/en/docs/Writing policies/jmespath.md [hunk 3]
```diff
@@ -2963,3 +2963,44 @@ spec:
...
+spec:
+  validationFailureAction: enforce
+  background: false
+  rules:
+    - name: validate-external-url
+      match:
+        any:
+      - resources:
+          kinds:
+            - ConfigMap
```
Comment (chipzoller) [inline content/en/docs/Writing policies/jmespath.md:2994]:
```suggestion
        - resources:
            kinds:
              - ConfigMap
```

File: content/en/docs/Writing policies/jmespath.md [hunk 4]
```diff
@@ -2963,3 +2963,44 @@ spec:
...
+      match:
+        any:
+        - resources:
+            kinds:
+              - ConfigMap
+      validate:
+        message: "ConfigMap contains an external URL."
+        deny:
+          conditions:
+           all:
```
Comment (chipzoller) [inline content/en/docs/Writing policies/jmespath.md:2999]:
```suggestion
            all:
```

File: content/en/docs/Writing policies/jmespath.md [hunk 5]
```diff
@@ -2963,3 +2963,44 @@ spec:
...
+
+**Example:** This policy looks for a field named `test` within the data section of ConfigMaps. It then runs a custom function called is_external_url to determine if the value associated with `test` is an external URL.
+
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: check-external-url-in-configmap
+spec:
+  validationFailureAction: enforce
```
Comment (chipzoller) [inline content/en/docs/Writing policies/jmespath.md:2986]:
```suggestion
  validationFailureAction: Enforce
```

**Pull request merged**